qadence/ml_tools/train_grad.py

```diff
@@ -122,8 +131,15 @@ def loss_fn(model: torch.nn.Module, data: torch.Tensor) -> tuple[torch.Tensor, d
         model = model.module.to(device=device, dtype=dtype)
     else:
         model = model.to(device=device, dtype=dtype)
-    # initialize tensorboard
-    writer = SummaryWriter(config.folder, purge_step=init_iter)
+    # initialize tracking tool
+    if config.tracking_tool == ExperimentTrackingTool.TENSORBOARD:
+        writer = SummaryWriter(config.folder, purge_step=init_iter)
+    else:
+        writer = importlib.import_module("mlflow")
+
+        # writer.mlflow.pytorch.autolog(
```

Comment:
Remove.

Reply:
should I then just import mlflow normally and set 
```python
else:
    writer = mlflow
```
?